build.sbt

```diff
@@ -1,2 +1,166 @@
 enablePlugins(akka.UnidocRoot, akka.TimeStampede, akka.UnidocWithPrValidation)
 disablePlugins(MimaPlugin)
+import com.typesafe.sbt.SbtMultiJvm.MultiJvmKeys.MultiJvm
+import com.typesafe.tools.mima.plugin.MimaPlugin
+import akka.AkkaBuild._
+
+initialize := {
+  System.setProperty("akka.mode", "test") // Is there better place for this? (and more importantly, what is it?)
```

Comment:
`git grep` says that the only other place where this is referenced is in the scaladoc of https://github.com/akka/akka/blob/master/akka-testkit/src/main/scala/akka/testkit/TestEventListener.scala#L487

This is not used anymore and safe to remove.